web-ui/src/main/resources/catalog/templates/admin/harvest/type/filesystem.js

```diff
@@ -53,7 +54,8 @@ var gnHarvesterfilesystem = {
                 + '    <checkFileLastModifiedForUpdate>' + h.site.checkFileLastModifiedForUpdate + '</checkFileLastModifiedForUpdate>'
                 + '    <directory>' + h.site.directory + '</directory>'
                 + '    <icon>' + h.site.icon + '</icon>'
-                + '  </site>' 
+                + '    <beforeScript>' + h.site.beforeScript + '</beforeScript>'
```

Comment:
I have no idea. I'm using the same pattern as `<directory>` which could suffer from the same problems. I don't know angularjs at all, so all i can do is mimic what's done around.
What would you do?